_pytest/_pluggy.py

```diff
@@ -1,11 +1,6 @@
 """
-imports symbols from vendored "pluggy" if available, otherwise
-falls back to importing "pluggy" from the default namespace.
+Import symbols from ``pluggy``
 """
 from __future__ import absolute_import, division, print_function
-try:
-    from _pytest.vendored_packages.pluggy import *  # noqa
-    from _pytest.vendored_packages.pluggy import __version__  # noqa
-except ImportError:
-    from pluggy import *  # noqa
-    from pluggy import __version__  # noqa
+from pluggy import *
+from pluggy import __version__
```

Comment:
we might want to consider just directly inlining the imports as a followup

Reply:
I think we can do it in the same PR if @tgoodlet is willing

Reply:
@nicoddemus @RonnyPfannschmidt yup no problem guys.
I'll take a look this morning.

Reply:
Awesome, thanks!